acq4/Manager.py

```diff
@@ -415,7 +415,7 @@ def loadDefinedConfig(self, name):
             if name not in self.config['configurations']:
                 raise Exception("Could not find configuration named '%s'" % name)
             cfg = self.config['configurations'].get(name, )
-        self.configure(cfg)
+        self.configure({'storageDir':cfg})
```

Comment:
You're right, I had my config file set up incorrectly. I changed the example config file to make it more clear how to set things up.